service/bfg/bfg.go

```diff
@@ -267,12 +259,16 @@ func (s *Server) handleRequest(parentCtx context.Context, bws *bfgWs, wsid strin
 	defer cancel()
 
 	select {
-	case <-s.requestLimiter:
+	case <-bws.requestLimiter:
```

Comment:
talking with @joshuasing ; it may be a good idea to _also_ have a global request limit too...need to think more about that 🤔